tests/api/tasks/by_entities_uris.test.ts

```diff
@@ -67,11 +21,19 @@ describe('tasks:bySuspectUris', () => {
     tasks[uri][0].should.be.an.Object()
   })
 
+  it('with empty type should return an array of tasks', async () => {
```

Comment:
[non-blocking] Keeping the "it should" form, unless too farfetched, reduces brain hick ups ;)
```suggestion
  it('should return an array of tasks when passed an empty type ', async () => {
```

Reply:
done in a2525a960